### DISCUSSION TITLE
Does omnetpp/inet supports parallel transmissions?

### DISCUSSION THREAD

**Discussion by marsluca** [omnetpp/omnetpp#1185] [General]
I have a network with four nodes with the same application. They should transmit simultaneously, but from the animation, it looks like they are taking turns. Do you know how to change this behavior? Thanks.

https://github.com/omnetpp/omnetpp/assets/34103646/389282d1-eb6c-4d8b-a45f-9ee3ef6c1838

**Comment by rhornig** [comment]
Yes it does. How the animation is displayed may or may not reflect the ordering of the packet transmissions in simulation time. Watch the simulation time display in the upper right. That is what matters. If that does not change then the time is not progressing so the packets are transmitting at the same time.

**Comment by marsluca** [comment]
Thank you very much for the response. Unfortunately, time continues to pass during the simulation. That's why I believe the transmission is not parallel.

https://github.com/omnetpp/omnetpp/assets/34103646/b8387014-7332-4402-9066-628b4c3c9fb6

**Reply by marsluca** [reply]
@rhornig

**Reply by rhornig** [reply]
indeed. These transmissions are not parallel. The hosts are definitely NOT starting the transmission at the same time. The fact that all the hosts have the same application does not mean that they will transmit at the same time. They may generate random numbers and base the transmission start on that random. So the question is, how the application is configured in each node. (i.e. what is written in the INI file).

**Reply by marsluca** [reply]
Thanks for your support. Indeed, there was a mistake in the INI file.
